pyproject.toml

```diff
@@ -2,11 +2,11 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "Cython",
+    "Cython<3",
```

Comment:
it turns out this change is not necessary anymore with the other changes in this commit.